packages/js/product-editor/src/components/button-with-dropdown-menu/index.tsx

```diff
@@ -29,8 +27,9 @@ export const ButtonWithDropdownMenu: React.FC<
 		offset: 0,
 	},
 	className,
+	renderMenu,
```

Comment:
Maybe we should modify the `button-with-dropdown-menu` [README.md](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/components/button-with-dropdown-menu/Readme.md#usage). We can add the `renderMenu` to the usage example and remove the `Add to store`. We can do this in a follow-up PR, though.